app/src/main/java/com/example/easycloset/ItemsAdapter.java

```diff
@@ -17,8 +18,10 @@
 import java.util.List;
 
 public class ItemsAdapter extends RecyclerView.Adapter<ItemsAdapter.ViewHolder> {
+
     private final Context context;
     private final List<Item> items;
+    private final int REQUEST_CODE = 7;
```

Comment:
so the idea to have named constant is to share it between places where you use it. If you do it then it is harder to break code by let's say updating number from 7 to 5 in one place and forget to do it in other. 
You need to move this to some file, you may call it "Constants.java" and then use it in both places.
As well, name is not very good since it is too generic, use something specific like "DELETE_ITEM_REQUEST_CODE"